### ISSUE TITLE
abhio_ L02E14

### ISSUE THREAD

**Issue by abhiojha8** [mikeudacity/robo-pace-betatest#55]
We currently provide the link for Linux cheat sheet if someone makes an error in submission

We can provide a link for Linux cheat sheet in the beginning of this page.

**Comment by stupsi99** [comment]
Agree, that cheat sheet should be in the beginning. Maybe even provide a solution if its not getting solved.
